uk_bin_collection/uk_bin_collection/councils/LondonBoroughOfRichmondUponThames.py

```
@@ -1,70 +1,164 @@
+            (s.get("value"), s.text.strip())
+            for s in streets
+            if s.get("value")
+        ]
+
+        if not street_usrns:
+            raise ValueError(f"No streets found for postcode {postcode}")
+
+        paon_lower = paon.strip().lower()
+
+        # Search each street for matching address
+        for usrn, street_name in street_usrns:
```

Ruff (0.15.15)

[warning] Loop control variable `street_name` not used within loop body

Rename unused `street_name` to `_street_name`

(B007)